lib/fetch/index.js

```diff
@@ -305,19 +305,17 @@ function finalizeAndReportTiming (response, initiatorType = 'other') {
   // global, and cacheState.
   markResourceTiming(
     timingInfo,
-    originalURL,
+    originalURL.href,
     initiatorType,
     globalThis,
     cacheState
   )
 }
 
 // https://w3c.github.io/resource-timing/#dfn-mark-resource-timing
-function markResourceTiming (timingInfo, originalURL, initiatorType, globalThis, cacheState) {
-  if (nodeMajor > 18 || (nodeMajor === 18 && nodeMinor >= 2)) {
-    performance.markResourceTiming(timingInfo, originalURL.href, initiatorType, globalThis, cacheState)
-  }
-}
+const markResourceTiming = (nodeMajor > 18 || (nodeMajor === 18 && nodeMinor >= 2))
```

Comment:
this seems highly unlikely to improve performance here, unless i'm missing something

Reply:
it removes an if from an hot path. Given it's not the main bottleneck   this is unlikely to yield massive improvements, but it's still something.